src/main/java/org/openstreetmap/atlas/geography/atlas/delta/AtlasDeltaGenerator.java

```diff
@@ -41,69 +42,57 @@ public AtlasDeltaGenerator(final Logger logger)
     @Override
     protected int onRun(final CommandMap command)
     {
-        final String baseName = (String) command.get("base");
-        final String alterName = (String) command.get("alter");
-        final File outputFolder = (File) command.get("outputFolder");
-        run(baseName, alterName, outputFolder);
+        final Path before = (Path) command.get("before");
+        final Path after = (Path) command.get("after");
+        final Path outputDir = (Path) command.get("outputDir");
+        run(before, after, outputDir);
         return 0;
     }
 
     @Override
     protected SwitchList switches()
     {
-        return new SwitchList().with(BASE_SWITCH, ALTER_SWITCH, OUTPUT_FOLDER_SWITCH);
+        return new SwitchList().with(BEFORE_SWITCH, AFTER_SWITCH, OUTPUT_DIR_SWITCH);
     }
 
-    private void compare(final String baseName, final Atlas base, final String alterName,
-            final Atlas alter, final String folder)
+    private void compare(final Atlas beforeAtlas, final Atlas afterAtlas, final Path outputDir)
     {
-        final AtlasDelta delta = new AtlasDelta(base, alter).generate();
-        final File diffFile = new File(folder)
-                .child(baseName + "_vs_" + alterName + FileSuffix.TEXT.toString());
-        diffFile.writeAndClose(delta.toString());
-        this.logger.info("Saved diff file {}", diffFile);
-        final File geoJsonFile = new File(folder)
-                .child(baseName + "_vs_" + alterName + FileSuffix.GEO_JSON.toString());
-        geoJsonFile.writeAndClose(delta.toGeoJson());
-        this.logger.info("Saved geoJson file {}", geoJsonFile);
-        final File relationsGeoJsonFile = new File(folder)
-                .child(baseName + "_vs_" + alterName + "_relations.geojson");
-        relationsGeoJsonFile.writeAndClose(delta.toRelationsGeoJson());
-        this.logger.info("Saved relationsGeoJsonFile file {}", relationsGeoJsonFile);
+        final String name = beforeAtlas.getName().split(".atlas")[0];
+
+        final AtlasDelta delta = new AtlasDelta(beforeAtlas, afterAtlas).generate();
+
+        final String txt = delta.toDiffViewFriendlyString();
```

Comment:
Nit: we try to avoid any and all abbreviations in `atlas`. We would prefer `text` over `txt`. If I didn't say it I know @MikeGost would have :)

Reply:
Sounds good, I'll change that!